rfc6962-bis.xml

```diff
@@ -1015,10 +1015,10 @@ an accepted root certificate.
 		  The Merkle Tree Hash of the tree, in base64.
 		</t>
 		<t hangText="tree_head_signature:">
-		  A TreeHeadSignature for the above data.
+		  A TreeHeadSignature for <spanx style="verb">tree_size</spanx>, <spanx style="verb">timestamp</spanx> and <spanx style="verb">sha256_root_hash</spanx>.
```

Comment:
Yes, I haven't really done this before.  I guess I should have made a separate branch for each from master? (Instead I branched after created the first PR).